tutorials/session-windows/ksql.html

```diff
@@ -0,0 +1,10 @@
+---
+layout: tutorial
+permalink: /create-session-windows/ksql
+stack: ksql
+static_data: session-windows
+
+try_it: tutorials/session-windows/ksql/markup/try_it.html
+test_it: tutorials/session-windows/ksql/markup/test_it.html
+take_it_to_prod: tutorials/session-windows/ksql/markup/take_it_to_prod.html
```

Comment:
Curious if these page variables are used?

Reply:
I think this is a hangover from last year when i first wrote the tutorial - I think the framework has changed a bit since then. I'll remove them.